live-examples/js-examples/expressions/expressions-nullishcoalescingoperator.html

```diff
@@ -0,0 +1,11 @@
+<pre>
+<code id="static-js">
+const foo = null ?? "default string";
+console.log(foo);
+// expected output: "default string"
+
+const bar = "" ?? "default string";
+console.log(bar);
+// expected output: ""
+</code>
+</pre>
```

Comment:
Can we add couple more examples to make sure users understand it correclty

false ?? true;   // => false
0 ?? 1;          // => 0
'' ?? 'default'; // => ''

null ?? [];      // => []
undefined ?? []; // => []

Reply:
It might be worth having more, but each one takes 4 lines, so to fit in a standard editor we can only have one more.

I'd suggest the one with numbers, that seems like a common case:

```js
const baz = 0 ?? 42;
console.log(baz);
// expected output: 0
```